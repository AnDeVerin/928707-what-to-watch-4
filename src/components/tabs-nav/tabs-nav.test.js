import React from 'react';
import renderer from 'react-test-renderer';
import TabsNav from './tabs-nav.jsx';

const mock = {
  tabs: [`tab 1`, `tab 2`],
  activeTab: `tab 1`,
  onClick: jest.fn(),
};

describe('TabsNav component', () => {
  it('renders correctly', () => {
    const { tabs, activeTab, onClick } = mock;

    const component = renderer
      .create(<TabsNav tabs={tabs} activeTab={activeTab} onClick={onClick} />)
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
