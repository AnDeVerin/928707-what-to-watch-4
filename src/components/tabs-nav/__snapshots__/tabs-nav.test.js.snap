// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabsNav component renders correctly 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
  >
    <li
      className="movie-nav__item movie-nav__item--active"
    >
      <a
        className="movie-nav__link"
        href="#"
        onClick={[Function]}
      >
        Tab 1
      </a>
    </li>
    <li
      className="movie-nav__item "
    >
      <a
        className="movie-nav__link"
        href="#"
        onClick={[Function]}
      >
        Tab 2
      </a>
    </li>
  </ul>
</nav>
`;
